hack/lib/buildx.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Check whether buildx exists.
+if ! docker buildx >/dev/null 2>&1; then
+  echo "ERROR: docker buildx not available. Docker 19.03 or higher is required with experimental features enabled"
+  exit 1
+fi
+
+# Use tonistiigi/binfmt that is to enable an execution of different multi-architecture containers
+docker run --privileged --rm tonistiigi/binfmt --install all
+
+# Create a new builder which gives access to the new multi-architecture features.
+builder_instance="build-node-sedna"
+temp=$(docker buildx ls | awk NF=1)
```

Comment:
How about using `docker buildx inspect $builder_instance`?

Reply:
https://github.com/docker/buildx/blob/master/docs/reference/buildx_inspect.md

Reply:
ok, use `if ! docker buildx inspect $build_instance`

Reply:
remember to discard stdout/stdin

Reply:
ok